sass/layout/_sidebar-left.scss

```diff
@@ -981,6 +980,7 @@
     .SidebarChannelLinkLabel_wrapper {
         overflow: hidden;
         display: flex;
+        margin-right: auto;
```

Comment:
`.SidebarChannelLinkLabel_wrapper` now pushes `.badge` and `. SidebarMenu` rightwards, this frees up margin-left on .badge to be used for "padding"

Reply:
Any reason we shouldn't use `flex-grow: 1` here instead of `margin-right: auto` since we're already using flexbox for layout? 0/5

Reply:
No, that's a good point. ✓